src/Core/tests/DeviceTests.Shared/HandlerTests/Focus/FocusHandlerTests.cs

```diff
@@ -58,8 +59,6 @@ public async Task FocusAndIsFocusedIsWorking()
 			});
 		}
 
-		// TODO: Android is not unfocusing
-#if IOS || MACCATALYST || WINDOWS
```

Comment:
At some point around API 30-ish focus behavior changed on Android and you can "unfocus" everything. We need to figure out where that happened and modify this test accordingly